app/models/container_image.rb

```diff
@@ -40,6 +47,12 @@ class ContainerImage < ApplicationRecord
 
   delegate :my_zone, :to => :ext_management_system, :allow_nil => true
 
+  PERF_ROLLUP_CHILDREN = [:containers]
+
+  def perf_rollup_parents(interval_name = nil)
+    [].compact unless interval_name == 'realtime'
```

Comment:
This is a little confusing, what are you trying to do here?  Return `nil` if realtime and `[]` otherwise?  Is the `.compact` necessary?
Maybe a comment clarifying what the `unless interval_name == 'realtime'` is doing (assuming it is because we only process rollups)